packages/dashboard/src/components/pages/platforms/components/instruments/instruments.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { TStringTranslation } from 'Types'
+
+const Instruments: React.FC<IInstrumentsType> = ({ title, active_symbols = [] }) => {
+    return (
+        <div className='dw-instruments'>
+            <Text className='dw-instruments__title' size='m' weight='bold'>
+                {title}
+            </Text>
+            {active_symbols.map((market, idx) => (
+                <React.Fragment key={idx}>
```

Comment:
Use the `market` name as a key rather than `idx`

Reply:
market name is not exactly an identifier, i would prefer call it market_index?

Reply:
See https://github.com/binary-com/deriv-app/pull/2476#discussion_r556974153

Reply:
`symbol`(eg: `frxAUDCHF`) is a unique identifier in `active_symbols`. Maybe we can use `market.symbol` instead of `idx`?